src/main/java/com/feelmycode/parabole/repository/EventParticipantRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.feelmycode.parabole.repository;
+
+
+import com.feelmycode.parabole.domain.EventParticipant;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+
+public interface EventParticipantRepository extends JpaRepository<EventParticipant,Long> {
+
+    Optional<EventParticipant> findByUserIdAndEventId(Long userId,Long eventId);
+
+
+
+}
```

Comment:
대체적으로 띄어쓰기가 많네욬ㅋㅋㅋㅋㅋㅋㅋ
```suggestion

import com.feelmycode.parabole.domain.EventParticipant;
import java.util.Optional;
import org.springframework.data.jpa.repository.JpaRepository;

public interface EventParticipantRepository extends JpaRepository<EventParticipant,Long> {

    Optional<EventParticipant> findByUserIdAndEventId(Long userId,Long eventId);
}
```